pkg/provisioner/controller.go

```diff
@@ -220,12 +221,18 @@ func (c *Controller) syncHandler(key string) error {
 func (c *Controller) handleProvisionClaim(key string, obc *v1alpha1.ObjectBucketClaim, class *storagev1.StorageClass) (err error) {
 
 	var (
-		ob        *v1alpha1.ObjectBucket
-		secret    *corev1.Secret
-		configMap *corev1.ConfigMap
+		ob               *v1alpha1.ObjectBucket
+		secret           *corev1.Secret
+		configMap        *corev1.ConfigMap
+		provisionerLabel = map[string]string{c.provisionerName: ""}
 	)
 	obcNsName := obc.Namespace + "/" + obc.Name
 
+	obc.ObjectMeta.SetLabels(provisionerLabel)
+	if obc, err = c.libClientset.ObjectbucketV1alpha1().ObjectBucketClaims(obc.Namespace).Update(obc); err != nil {
```

Comment:
I wish this update can be called once for the combined update of the labels, the new finalizer ([protectOBC](https://github.com/kube-object-storage/lib-bucket-provisioner/pull/139/files#diff-5d07bb1605a6a9c2d1308ba7348be5f3R253) from #139 and the phase update to "pending".

Reply:
Unfortunately, we can only get it down to a min of 2 calls.  UpdateStatus is exclusive to object status updates whereas Update only applies changes to Spec and Metadata.   We can condense the finalizer and label updates once this PR or #139 is merged, then make the changes in the other.